app/controllers/global_activities_controller.rb

```diff
@@ -57,8 +61,16 @@ def search_subjects
   private
 
   def activity_filters
+    begin
+      params[:types] = JSON.parse(params[:types])
+      params[:users] = JSON.parse(params[:users])
+      params[:teams] = JSON.parse(params[:teams])
+      params[:subjects] = JSON.parse(params[:subjects])
+    rescue StandardError
+    end
```

Comment:
Why  this?

Reply:
I mean: Rescuing from StandardError without any action can make debugging unpleasant :). And what could possibly happen here that needs this kind of error handling?

Reply:
Theoretically this will never raise error cause we are creating all arrays... currently i don't know what kind of error handling can be performed here.